workflowwebtools/workflowinfo.py

```diff
@@ -385,7 +385,8 @@ def site_to_run(self, task):
         for site in site_set:
             clean_site = re.sub(r'_(ECHO_)?(Disk|MSS)$', '', site)
             if clean_site not in out_list and clean_site and \
-                    clean_site in all_site_list:
+                    clean_site in all_site_list and \
+                    'T0_' not in clean_site:
```

Comment:
what if in the future we get T0_* in production. I think the issue is that these are getting added because the MSS or Export endpoint is present as a location

Reply:
The issue by Sharad just says "T0_CH_CERN" is being assigned when it shouldn't. It doesn't look like that includes MSS or Export. (But I don't know off the top of my head if that's the full name.)

I absolutely agree that this is the wrong place to put a filter like this. That's why I say above this was originally supposed to be a temporary fix. However the other filters don't seem to be working.

Reply:
you have to change 

clean_site = re.sub(r'_(ECHO_)?(Disk|MSS)$', '', site) 

for 

clean_site = re.sub(r'_(ECHO_)?(Disk)$', '', site)

or do anything that will remove any *_MSS *_Export stuff from the whitelist send by the console.

Reply:
I'm not sure what particular problem you are trying to fix with your suggestion. The suffixes are already stripped from anything that is sent from the console. That's what those lines are doing. Will that change fix https://github.com/CMSCompOps/wtc-console/issues/5? What's the full name of T0_CH_CERN?

Reply:
if one of the data location is T<anything>_MSS or T<anything>_Export, that data location has to be completely ignored, and not make through in the list of site suggested by the console as where to run the acdc

Reply:
looks like if you have T1_IT_CNAF_MSS, one will get  T1_IT_CNAF suggested, and all the way to the acdc configuration

Reply:
Yeah, that will happen. I can take that out.

Reply:
I'll open another PR with that change.